docs/visual-basic/language-reference/statements/if-then-else-statement.md

```diff
@@ -93,17 +103,19 @@ If condition Then [ statements ] [ Else [ elsestatements ] ]
   
  In the single-line syntax, you can have multiple statements executed as the result of an `If`...`Then` decision. All statements must be on the same line and be separated by colons.  
   
-## Example  
+<a name="multi-line"></a>
+ 
  The following example illustrates the use of the multiple-line syntax of the `If`...`Then`...`Else` statement.  
   
  [!code-vb[VbVbalrStatements#101](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_1.vb)]  
   
-## Example  
+<a name="nested"></a>
+
  The following example contains nested `If`...`Then`...`Else` statements.  
   
  [!code-vb[VbVbalrStatements#102](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_2.vb)]  
   
-## Example  
+<a name="single-line"></a>
  The following example illustrates the use of the single-line syntax.  
   
  [!code-vb[VbVbalrStatements#103](../../../visual-basic/language-reference/error-messages/codesnippet/VisualBasic/if-then-else-statement_3.vb)]  
```

Comment:
Here again, this could be a complete code example that's executable. You can use highlighting here (as well as in the other two examples) to focus on the If statements.